modules/cnf-topology-aware-lifecycle-manager-troubleshooting.adoc

```diff
@@ -435,3 +433,13 @@ ERROR	controller-runtime.manager.controller.clustergroupupgrade	Reconciler error
 sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
 ----
 <1> Displays the error.
+
+[discrete]
+=== Clusters are not compliant to some policies after a `ClusterGroupUpgrade` CR has completed
+
+Issue:: The compliance status of the policy has not yet updated. 
+This may be because:
+* The CGU was run too soon after a policy was created or updated. 
+* The remediation of a policy affects the compliance of subsequent policies in the `ClusterGroupUpgrade` CR.
+
+Resolution:: Delete or rename the `ClusterGroupUpdate` CR and apply it again.
```

Comment:
Rename is a bit misleading. Existing CR can't be renamed. Maybe drop the delete part as it's good to keep it a reference if they want to check what happened to the first attempt. Just say create/apply a new CR with the same spec.

Reply:
done